Source/Core/DolphinQt/Settings/AudioPane.h

```diff
@@ -35,6 +37,9 @@ class AudioPane final : public QWidget
   void CheckNeedForLatencyControl();
   bool m_latency_control_supported;
 
+  const QString GetDPL2QualityLabel(AudioCommon::DPL2Quality value);
```

Comment:
By const, I meant

```suggestion
QString GetDPL2QualityLabel(AudioCommon::DPL2Quality value) const;
```

const on a function returning by value doesn't really do much (other than potentially inhibit copy-elision)